src/extension.ts

```diff
@@ -20,4 +21,9 @@ export function activate(context: vscode.ExtensionContext) {
 	return commonActivate(context, view);
 }
 
-export function deactivate() { }
+export async function deactivate() {
+	// Do our best to log out of all sessions
+	for (const serverSession of serverSessions) {
```

Comment:
Should this use a `Promise.allSettled()` approach like the vscode-objectscript PR to make the REST requests in parallel?

Reply:
Good suggestion. Now done.